src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.UnitTests/TelemetryReporterTests.cs

```diff
@@ -44,13 +45,13 @@ public async Task TestBlockLogging()
     {
         var service = await CreateReporterAsync();
         service.LogBlockStart(GetEventName(nameof(TestBlockLogging)), kind: 0, blockId: 0);
-        service.LogBlockEnd(blockId: 0, ImmutableDictionary<string, object?>.Empty, CancellationToken.None);
+        service.LogBlockEnd(blockId: 0, [], CancellationToken.None);
```

Comment:
Unclear if this is better.  this will always allocate a new list instance.  is that really what is intended?

could this intead allow null?

Reply:
this is in test code, was there a product code aspect of this concern?

Reply:
oh.  my bad.  never mind that feedback then.  let me recheck this.